src/Workspaces/Core/Portable/PublicAPI.Shipped.txt

```diff
@@ -840,7 +840,6 @@ abstract Microsoft.CodeAnalysis.CodeFixes.CodeFixProvider.RegisterCodeFixesAsync
 abstract Microsoft.CodeAnalysis.CodeFixes.FixAllContext.DiagnosticProvider.GetAllDiagnosticsAsync(Microsoft.CodeAnalysis.Project project, System.Threading.CancellationToken cancellationToken) -> System.Threading.Tasks.Task<System.Collections.Generic.IEnumerable<Microsoft.CodeAnalysis.Diagnostic>>
 abstract Microsoft.CodeAnalysis.CodeFixes.FixAllContext.DiagnosticProvider.GetDocumentDiagnosticsAsync(Microsoft.CodeAnalysis.Document document, System.Threading.CancellationToken cancellationToken) -> System.Threading.Tasks.Task<System.Collections.Generic.IEnumerable<Microsoft.CodeAnalysis.Diagnostic>>
 abstract Microsoft.CodeAnalysis.CodeFixes.FixAllContext.DiagnosticProvider.GetProjectDiagnosticsAsync(Microsoft.CodeAnalysis.Project project, System.Threading.CancellationToken cancellationToken) -> System.Threading.Tasks.Task<System.Collections.Generic.IEnumerable<Microsoft.CodeAnalysis.Diagnostic>>
-abstract Microsoft.CodeAnalysis.CodeFixes.FixAllProvider.GetFixAsync(Microsoft.CodeAnalysis.CodeFixes.FixAllContext fixAllContext) -> System.Threading.Tasks.Task<Microsoft.CodeAnalysis.CodeActions.CodeAction>
```

Comment:
This makes the public API inconsistent with our other public APIs for analyzers and fixers. All our diagnostic analysis contexts and code fix/refactoring context objects have a cancellation token, and this token is exposed directly on the context. For example, see [RegisterCodeFixesAsync](http://source.roslyn.io/Microsoft.CodeAnalysis.Workspaces/R/58e334d289454f55.html) and the [CancellationToken](http://source.roslyn.io/Microsoft.CodeAnalysis.Workspaces/R/c57f5d43f0e76de4.html) exposed by it. It was an explicit decision by the API design team to keep the FixAllContext consistent with other context objects.